backends-clickhouse/src-iceberg/test/scala/org/apache/gluten/execution/iceberg/ClickHouseIcebergSuite.scala

```diff
@@ -303,7 +303,7 @@ class ClickHouseIcebergSuite extends GlutenClickHouseWholeStageTransformerSuite
   }
 
   testWithSpecifiedSparkVersion(
-    "iceberg bucketed join partition value not exists partial cluster", Array("3.4", "3.5")) {
+    "iceberg bucketed join partition value not exists partial cluster", "3.4", "3.5") {
```

Comment:
nit: this can also be changed to `testWithMinSparkVersion`

Reply:
thanks, is should be all.
![image](https://github.com/user-attachments/assets/96926b0c-33c5-49d3-82a9-e2de94b51bec)